src/data/record.py

```diff
@@ -1,7 +1,11 @@
+from pyproj import Transformer
 from . import util
 from dateutil.parser import parse
 
 
+transformer = Transformer.from_proj(4326, 3857, always_xy=True)
```

Comment:
@j-t-t , do you think it makes sense to establish the transformer here and then just import it where you import the Record class? It's a minor thing, but maybe makes sense over establishing the transformer across a few different scripts.

Reply:
Yeah, that makes sense. I've updated record.py to include the two transformers we need (one for each projection direction).